portality/reapplication.py

```diff
@@ -482,6 +482,12 @@ def other_list(main_field, other_field, other_value):
             aids = forminfo.get(main_field, [])
             if aids is None or aids == "" or aids == "None":
                 aids = []
+
+            # if the xwalk has returned a single-list element like ["None"]
+            # we want to strip that "None" for the purpose of the CSV
+            if choices.Choices.NONE in aids:
+                aids.remove(choices.Choices.NONE)
```

Comment:
It is acceptable to do this on a generic basis here in the function instead of per question, because it's only article_identifiers, deposit_policies (both asked to be blanked in case of negative answer in #525) and fulltext_formats that use this function. fulltext_formats will not be affected by this change - it's a list question like the others and it's acceptable to blank it in the absence of data.